pkgs/tools/misc/lesspipe/default.nix

```diff
@@ -1,35 +1,35 @@
-{ lib, stdenv, fetchFromGitHub, substituteAll, perl, file, ncurses, bash }:
+{ lib, stdenv, fetchFromGitHub, substituteAll, makeWrapper, perl, procps, file, gnused, bash }:
 
 stdenv.mkDerivation rec {
   pname = "lesspipe";
-  version = "1.85";
+  version = "2.05";
 
-  nativeBuildInputs = [ perl ];
+  nativeBuildInputs = [ perl makeWrapper ];
   buildInputs = [ perl bash ];
   strictDeps = true;
   preConfigure = ''
     patchShebangs --build configure
+    substituteInPlace configure --replace '/etc/bash_completion.d' '/share/bash-completion/completions'
   '';
-  configureFlags = [ "--shell=${bash}/bin/bash" "--yes" ];
-  configurePlatforms = [];
+  configureFlags = [ "--shell=${bash}/bin/bash" "--prefix=/" ];
+  configurePlatforms = [ ];
   dontBuild = true;
 
+  installFlags = [ "DESTDIR=$(out)" ];
+
+  postInstall = ''
+    for f in lesspipe.sh lesscomplete; do
+      wrapProgram "$out/bin/$f" --prefix-each PATH : "${lib.makeBinPath [ file gnused procps ]}"
+    done
+  '';
+
   src = fetchFromGitHub {
```

Comment:
Can we please change the sorting to be more standard?

Reply:
yes

Reply:
Done.